Packs/Core/ReleaseNotes/2_0_2.md

```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+
+##### Investigation & Response
+
+- Fixed an issue where the integration failed when searching for a value that does exist, when running one of the following commands:
+    - ***core-list-risky-users***
+    - ***core-list-risky-hosts***
+    - ***core-list-user-groups***
+    - ***core-list-roles***
```

Comment:
```suggestion
Fixed an issue where the integration failed when searching for a value exists, when running one of the following commands:
- ***core-list-risky-users***
- ***core-list-risky-hosts***
- ***core-list-user-groups***
- ***core-list-roles***
```

Reply:
this is typo issue, i mean *doesn't*  exist